PyBASC/basc.py

```diff
@@ -98,7 +98,7 @@ def group_dim_reduce(
 def nifti_individual_stability(
     subject_file, roi_mask_file,
     n_bootstraps, n_clusters, compression_dim, similarity_metric,
-    compressor=False, cross_cluster=False, cxc_compressor=False, cxc_roi_mask_file=None,
+    compressor=None, cross_cluster=False, cxc_compressor=None, cxc_roi_mask_file=None,
```

Comment:
Double check this is changed in all the other places compressor is called, and created.

in Group_dim_reduce compressor is initially set if groupDimreduce=True

Reply:
Yes! The check will actually be the same as if it is `False`:
```
if compressor:
```
because `bool(None)` is `False`. They are now `None` or `FeatureAgglomeration` object